src/test/java/org/pesmypetcare/webservice/builders/PathTest.java

```diff
@@ -82,13 +82,15 @@ public void buildUsersCollectionPath() {
     @Test
     public void buildForumPath() {
         String path = Path.ofDocument(Collections.forums, groupId, forumId);
-        assertEquals("groups/" + groupId + "/forums/" + forumId, path, "Should return the path to the forum.");
+        assertEquals("groups/" + groupId + "/forums/" + forumId, path,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "/forums/" appears 4 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5482446)